com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/view/SQLTableManageView.java

```diff
@@ -610,15 +611,22 @@ public void run(IProgressMonitor monitor)
         }
     }
 
-    /** setup real combo box value(target schema name list) */
+    /** setup combo box value(target schema name list) */
     protected void setupCombobox() {
-        String[] schemaNameArr =
-                config.getTarCatalog().getSchemas().stream()
-                        .map(Schema::getName)
-                        .toArray(String[]::new);
-        this.tarSchemaList = schemaNameArr;
+        if (config.isAddUserSchema()) {
+            Catalog catalog = (config.getTarCatalog() != null)
```

Comment:
소괄호가 꼭 필요해 보이지는 않습니다.
```suggestion
            Catalog catalog = config.getTarCatalog() != null
```

Reply:
확인했습니다 수정사항 적용하겠습니다.